views/chitter/index.erb

```diff
@@ -0,0 +1,30 @@
+<ul>
```

Comment:
OK, looking at this I see why you've gathered a few different collections of peeps in your controller action.  you could simplify this template, by making it agnostic of the ordering, if you add some equivalent logic to your controller and / or Peep class.  For example, imagine you have a checkbox called `sort_by_newest_first`.  Then...

* In your controller you'd check the state of the checkbox
* If it's checked, you call something like Peep.all_reverse_chronological
* Else, you'd call Peep.all